code/modules/clothing/gloves/color.dm

```diff
@@ -142,7 +142,7 @@
 	item_color = "cargo"					//Exists for washing machines. Is not different from brown gloves in any way.
 
 /obj/item/clothing/gloves/color/captain
-	desc = "Regal blue gloves, with a nice gold trim. Swanky."
+	desc = "Regal blue gloves, with a nice gold trim, diamond anti-shock coating, and integrated thermal barrier. Swanky."

```

Comment:
trim, coating, and barrier should all have the same article, be it 'a' or nothing.